.web-extension-id

```diff
@@ -0,0 +1,3 @@
+# This file was created by https://github.com/mozilla/web-ext
```

Comment:
No, this is needs to be persisted because otherwise it'll always be generated and result in a new extension instead of new version to existing one.